image-specs/el7.3-base

```diff
@@ -0,0 +1,16 @@
+#base=libguestfs:centos-7.3
```

Comment:
1. It will allow to rollback to the old version (if needed).
2. If we won't change the name, Lago will keep using the old image from the local cache.
3. Personally it think it's better to use another name so we can infer from lago.log that the   
    new image is being used.